src/components/form/from.js

```diff
@@ -1,17 +1,57 @@
 import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
 import Button from '../button/button';
+import { addBook } from '../../redux/books/books';
+
+const isValid = (state) => state && state !== 'Category';
 
 export default function Form() {
   const [bookTitle, setBookTitle] = useState('');
   const [selected, setSelected] = useState('');
+  const [valid, setValid] = useState([true, true]);
+  const dispatch = useDispatch();
+
   const handleChange = (e) => {
     setBookTitle(e.target.value);
   };
+
```

Comment:
Please ensure that your `New Book React Component` has a local state with `title`, `author` according to the project requirement.

![image](https://user-images.githubusercontent.com/10332499/160614299-3d207e8e-fdef-474f-9a16-2c43e2605850.png)

Reply:
Thank you for the clarification :pray: